Node CI not passing compilation issue

- jars have been already upgraded to the latest version required

stil:

```
Using Kotlin/JVM incremental compilation
[KOTLIN] Kotlin compilation 'jdkHome' argument: /opt/hostedtoolcache/Java_Zulu_jdk/21.0.10-7/x64
Options for KOTLIN DAEMON: IncrementalCompilationOptions(super=CompilationOptions(compilerMode=INCREMENTAL_COMPILER, targetPlatform=JVM, reportCategories=[0, 3], reportSeverity=2, requestedCompilationResults=[0], kotlinScriptExtensions=[]), sourceChanges=org.jetbrains.kotlin.buildtools.api.SourcesChanges$Unknown@64accb41, classpathChanges=NotAvailableForNonIncrementalRun, workingDir=/home/runner/work/bisq-mobile/bisq-mobile/apps/nodeApp/build/kotlin/compileDebugKotlinAndroid/cacheable, multiModuleICSettings=MultiModuleICSettings(buildHistoryFile=/home/runner/work/bisq-mobile/bisq-mobile/apps/nodeApp/build/kotlin/compileDebugKotlinAndroid/local-state/build-history.bin, useModuleDetection=true), icFeatures=IncrementalCompilationFeatures(usePreciseJavaTracking=true, withAbiSnapshot=false, preciseCompilationResultsBackup=true, keepIncrementalCompilationCachesInMemory=true, enableUnsafeIncrementalCompilationForMultiplatform=false, enableMonotonousIncrementalCompileSetExpansion=true), outputFiles=[/home/runner/work/bisq-mobile/bisq-mobile/apps/nodeApp/build/tmp/kotlin-classes/debug, /home/runner/work/bisq-mobile/bisq-mobile/apps/nodeApp/build/kotlin/compileDebugKotlinAndroid/cacheable, /home/runner/work/bisq-mobile/bisq-mobile/apps/nodeApp/build/kotlin/compileDebugKotlinAndroid/local-state])
Finished executing kotlin compiler using DAEMON strategy
e: file:///home/runner/work/bisq-mobile/bisq-mobile/apps/nodeApp/src/androidMain/kotlin/network/bisq/mobile/node/common/domain/service/accounts/NodeFiatAccountsServiceFacade.kt:60:28 Unresolved reference 'updatePaymentAccount'.

e: file:///home/runner/work/bisq-mobile/bisq-mobile/apps/nodeApp/src/androidMain/kotlin/network/bisq/mobile/node/main/NodeMainApplication.kt:189:26 Unresolved reference 'setup'.
e: file:///home/runner/work/bisq-mobile/bisq-mobile/apps/nodeApp/src/androidMain/kotlin/network/bisq/mobile/node/main/NodeMainApplication.kt:190:25 Unresolved reference 'setup'.
> Task :apps:nodeApp:compileReleaseKotlinAndroid FAILED
```

it might be the new deps gradle caching rule?

also might be good to check kover error

```
> Task :shared:presentation:koverVerify
Caching disabled for task ':shared:presentation:koverVerify' because:
  Caching has been disabled for the task
Task ':shared:presentation:koverVerify' is not up-to-date because:
  Task.upToDateWhen is false.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':koverVerify'.
> Rule 'Check Coverage' violated: lines covered percentage is 64.872700, but expected minimum is 65


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org./

BUILD FAILED in 1m 31s

> Task :koverCachedVerify
Build cache key for task ':koverCachedVerify' is e3366632265939a507dfc5440733d981
Task ':koverCachedVerify' is not up-to-date because:
  No history is available.
Stored cache entry for task ':koverCachedVerify' with cache key e3366632265939a507dfc5440733d981
Resolve mutations for :koverVerify (Thread[#283,included builds,5,main]) started.
:koverVerify (Thread[#283,included builds,5,main]) started.

> Task :koverVerify FAILED
Caching disabled for task ':koverVerify' because:
  Caching has been disabled for the task
Task ':koverVerify' is not up-to-date because:
  Task.upToDateWhen is false.
Build 98fe30ef-cfcd-4e04-a276-e451ffdb7121 is closed
```

in this one https://github.com/bisq-network/bisq-mobile/actions/runs/21510618913/job/61976492945?pr=1112

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -31,7 +31,7 @@ jobs:
           path: |
             ~/.gradle/caches
             ~/.gradle/wrapper
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties', '**/libs.versions.toml') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
 
@@ -76,7 +76,7 @@ jobs:
           path: |
             ~/.gradle/caches
             ~/.gradle/wrapper
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties', '**/libs.versions.toml') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
 
@@ -86,7 +86,7 @@ jobs:
         with:
           path: |
             ~/.konan
-          key: ${{ runner.os }}-konan-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          key: ${{ runner.os }}-konan-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties', '**/libs.versions.toml') }}
           restore-keys: |
             ${{ runner.os }}-konan-
 
@@ -276,7 +276,7 @@ jobs:
           path: |
             ~/.gradle/caches
             ~/.gradle/wrapper
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties', '**/libs.versions.toml') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
 

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -2,7 +2,8 @@
 
 # -------------------- Project Specific --------------------
 bisq-core = "2.1.8"
-bisq-core-commit = "2428db01510b3fa6f5290a6cd3b805d91580e0ec"
+# Update this commit hash when bisq2 JARs are regenerated (even with same version) to bust CI cache
+bisq-core-commit = "96a07890a2c65c5e98f9525ad92282bc4fac04ad"
 # API version is usually same as bisq-core but for more control we keep it separated
 #bisq-api = "2.1.8"
 

